### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by ozbwrk** [community/community#27293] [Other Feature Feedback, Questions, & Ideas] [answered]
README is not showing at the top in my profile how can I show.

**Comment by jfullstackdev** [comment, marked as answer]
you need to create a repo named `ozberkmatreas`, that is a special repo that will be displayed on your landing page with the README, like mine

![image](https://user-images.githubusercontent.com/47092464/181242845-4b22216e-ca10-44e2-bfb8-60c0c49837bc.png)

**Reply by ozbwrk** [reply]
How to make special repo ?

**Reply by airtower-luna** [reply]
Like any other repo, what makes it special is that the name is the same as your username. :wink:

**Reply by jfullstackdev** [reply]
create a new repo, with that name exactly as your username

**Reply by ozbwrk** [reply]
oky 1 minutes

**Reply by ozbwrk** [reply]
ooo thanks bro

**Reply by qiewi** [reply]
hello, ive made my special repo but it does not show up on my profile. Anyone knows why?

**Reply by vLecaBR** [reply]
great
